test/e2e/suite_test.go

```diff
@@ -344,7 +342,7 @@ func beforeTest(t *testing.T, f *framework.E2eFramework, fixtureDir string, opts
 		return err
 	}
 
-	if err := f.WaitForCassOperatorToBeReady(cassOperatorNS, polling.operatorDeploymentReady.timeout, polling.operatorDeploymentReady.interval); err != nil {
+	if err := f.WaitForCassOperatorToBeReady(opts.sutNamespace, polling.operatorDeploymentReady.timeout, polling.operatorDeploymentReady.interval); err != nil {
```

Comment:
Shouldn't this be `opts.operatorNamespace`?

Reply:
You're right, thanks for calling this out!